lib/Server.js

```diff
@@ -52,6 +53,15 @@ function Server(compiler, options) {
   const invalidPlugin = () => {
     this.sockWrite(this.sockets, 'invalid');
   };
+  if (this.progress) {
+    const progressPlugin = new webpack.ProgressPlugin((percent, msg, addInfo) => {
+      percent = Math.floor(percent * 100);
+      if (percent === 100) msg = 'Compilation competed';
```

Comment:
`Compilation competed` -> `Compilation completed`

Reply:
I'll clean up that typo in a bit